dts/msm8916/msm8939-huawei-kiwi.dts

```diff
@@ -16,4 +16,7 @@
 
 	model = "Huawei Honor 5X";
 	compatible = "huawei,kiwi", "qcom,msm8939", "lk2nd,device";
+
+	// fix modem not connecting to cellular network
+	huawei,modem_id = <0xFFB40105 0x0>;
```

Comment:
The downstream kernel seems to have several different of these "modem_id"s so I'm not sure if we should just hardcode one particular one here. Might need to split this up in multiple device trees. I think someone had an opinion on that in the postmarketOS mainline chat but iirc they don't have a GitHub account...